.flutter-plugins-dependencies

```diff
@@ -1 +1 @@
ds\\\\flutter_windows_2.5.2-stable\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\firebase_core-1.7.0\\\\","dependencies":[]},{"name":"geolocator_apple","path":"C:\\\\Users\\\\131la\\\\Downloads\\\\flutter_windows_2.5.2-stable\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\geolocator_apple-1.2.2\\\\","dependencies":[]},{"name":"sqflite","path":"C:\\\\Users\\\\131la\\\\Downloads\\\\flutter_windows_2.5.2-stable\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\sqflite-2.0.0+4\\\\","dependencies":[]}],"linux":[],"windows":[],"web":[{"name":"cloud_firestore_web","path":"C:\\\\Users\\\\131la\\\\Downloads\\\\flutter_windows_2.5.2-stable\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\cloud_firestore_web-2.4.3\\\\","dependencies":["firebase_core_web"]},{"name":"firebase_auth_web","path":"C:\\\\Users\\\\131la\\\\Downloads\\\\flutter_windows_2.5.2-stable\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\firebase_auth_web-3.1.2\\\\","dependencies":["firebase_core_web"]},{"name":"firebase_core_web","path":"C:\\\\Users\\\\131la\\\\Downloads\\\\flutter_windows_2.5.2-stable\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\firebase_core_web-1.1.0\\\\","dependencies":[]},{"name":"flutter_facebook_auth_web","path":"C:\\\\Users\\\\131la\\\\Downloads\\\\flutter_windows_2.5.2-stable\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\flutter_facebook_auth_web-2.6.0+2\\\\","dependencies":[]},{"name":"geolocator_web","path":"C:\\\\Users\\\\131la\\\\Downloads\\\\flutter_windows_2.5.2-stable\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\geolocator_web-2.0.6\\\\","dependencies":[]},{"name":"google_maps_flutter_web","path":"C:\\\\Users\\\\131la\\\\Downloads\\\\flutter_windows_2.5.2-stable\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\google_maps_flutter_web-0.3.2\\\\","dependencies":[]}]},"dependencyGraph":[{"name":"cloud_firestore","dependencies":["cloud_firestore_web","firebase_core"]},{"name":"cloud_firestore_web","dependencies":["firebase_core","firebase_core_web"]},{"name":"firebase_auth","dependencies":["firebase_auth_web","firebase_core"]},{"name":"firebase_auth_web","dependencies":["firebase_core","firebase_core_web"]},{"name":"firebase_core","dependencies":["firebase_core_web"]},{"name":"firebase_core_web","dependencies":[]},{"name":"flutter_facebook_auth","dependencies":["flutter_facebook_auth_web"]},{"name":"flutter_facebook_auth_web","dependencies":[]},{"name":"flutter_plugin_android_lifecycle","dependencies":[]},{"name":"geolocator","dependencies":["geolocator_android","geolocator_apple","geolocator_web"]},{"name":"geolocator_android","dependencies":[]},{"name":"geolocator_apple","dependencies":[]},{"name":"geolocator_web","dependencies":[]},{"name":"google_maps_flutter","dependencies":["flutter_plugin_android_lifecycle"]},{"name":"google_maps_flutter_web","dependencies":[]},{"name":"sqflite","dependencies":[]}],"date_created":"2021-10-17 10:34:38.638945","version":"2.5.2"}
```

Comment:
In future please do not commit this, or other configuration files to the repository.

**TODO**
- Add .flutter-plugins-dependencies to .gitignore